backstop/tests/tile.html

```diff
@@ -1060,5 +1060,231 @@ <h2>Folder</h2>
         </div>
       </div>
     </section>
+
+    <hr />
+
+    <h2>Responsive array of tiles</h2>
+    <section id="demo-tile-array">
+
+      <div class="iui-tile-array">
+        <div
+          class="iui-tile"
+          tabindex="0"
+        >
+          <div class="iui-tile-thumbnail">
+            <svg-imodel-hollow
+              class="iui-thumbnail-icon"
+              aria-hidden="true"
+            ></svg-imodel-hollow>
+          </div>
+
+          <div class="iui-tile-content ">
+            <div class="iui-tile-name">
+              <span class="iui-tile-name-label">Tile 1</span>
+            </div>
+            <div class="iui-tile-description">
+              Bentley Systems, Incorporated, is an American-based software
+              development company that develops, manufactures, licenses, sells and
+              supports computer software and services for the design,
+              construction, and operation of infrastructure.
+            </div>
+          </div>
+        </div>
+
+        <div
+          class="iui-tile"
+          tabindex="0"
+        >
+          <div class="iui-tile-thumbnail">
+            <svg-imodel-hollow
```

Comment:
Can you try to use image instead of SVG? Because I have concerns about this width:
![image](https://user-images.githubusercontent.com/36186912/153812620-8910cf23-762a-4df7-87a1-f1f62d43d802.png)

Reply:
I've added photos to every other tile, everything appears good.

Reply:
Can you also add the `<img>` tag to some of the tiles? It is a supported way of showing tile thumbnails.

Reply:
Tile 4 & 8 within the responsive array section are using `<img>`.  Had to add this to the CSS:
```scss
@at-root {
  img#{&} {
    object-fit: cover;
  }
}
```